src/test/java/mate/academy/lib/InjectorTest.java

```diff
@@ -14,7 +14,7 @@
 import java.util.Arrays;
 
 class InjectorTest {
-    private Injector injector = Injector.getInjector();
+    private final Injector injector = Injector.getInjector();
```

Comment:
Why final? I don't understand

Reply:
This is InjectorTest class
I didn't implement it